CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning], with the exception that minor rel
 - ✨ Add translation from `QuantumComputation` to the `MQTRef` MLIR dialect ([#1099]) ([**@denialhaag**], [**@burgholzer**])
 - ✨ Add `reset` operations to the MLIR dialects ([#1106]) ([**@DRovara**])
 - ✨ Add MQT's implementation of a QDMI Device for neutral atom-based quantum computing ([#996], [#1010]) ([**@ystade**])
+- ✨ Add new MLIR pass `ElidePermutations` for SWAP gate elimination ([#1140]) ([**@taminob**])
```

Comment:
Should be moved to the top of the list. (although we might really have to rethink this at some point. Apparently it feels more natural for people to extend the list at the bottom)

Reply:
It was me twice, but yes, it feels more intuitive to extend a list at the end. But once you know that new entries should be added to the front, that is more convenient since you need to scroll less if the list is already pretty long.